src/api/app/components/canned_responses_dropdown_component.html.haml

```diff
@@ -0,0 +1,15 @@
+.dropdown
+  %button.btn.btn-link.text-decoration-none.dropdown-toggle{ type: 'button', 'data-bs-toggle': 'dropdown', 'aria-expanded': false }
+    %i.fas.fa-comment-dots
+    Canned Responses
+  %ul.dropdown-menu.dropdown-menu-end
+    - @canned_responses.each do |canned_response|
+      %li.dropdown-item{ data: { 'canned-response': canned_response.content } }= canned_response.title
```

Comment:
It would be nice to have on these elements the usual cursor for clickable elements with `cursor: pointer`.